React/finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,28 @@
+import isArray from './utils';
+
+export function min(...data) {
+  if (data[0] === undefined) {
+    return (data[0]);
+  } else if (isArray(...data)) {
+    return (Math.min(...data[0]));
+  } else if (isArray(data)) {
+    return (Math.min(...data));
+  }
+}
+
+export function copy(...data) {
+  if (isArray(...data)) {
+    return [...data[0]];
+  } else {
+    return { ...data[0] };
+  }
```

Comment:
this function can be one line... try to think of a way to do it. Hint: ternary operator